power_grid_model_c/power_grid_model_c/include/power_grid_model_c.h

```diff
@@ -91,8 +93,10 @@ enum PGM_CalculationMethod {
     PGM_newton_raphson = 1,    /**< Newton-Raphson method for power flow */
     PGM_iterative_linear = 2,  /**< iterative linear method for state estimation */
     PGM_iterative_current = 3, /**< linear current method for power flow */
-    PGM_linear_current = 4     /**< iterative constant impedance method for power flow */
+    PGM_linear_current = 4,    /**< iterative constant impedance method for power flow */
+    PGM_iec60909 = 5           /**< sequential fault analysis for short circuits */
 };
```

Comment:
I think it would be appropriate to add a default calculation method enum: `PGM_default_method`. In the CPP file of `PGM_Options` definition, `PGM_default_method` should be the default value for `calculation_method`.

Reasoning:
1. Different calculation types require different method. We now default the method to `newton_raphson` which is only valid for power flow. If user runs short circuit or state estimation via C-API, he always has to specify a valid method, otherwise it will crash.
2. By add a new enumeration `PGM_default_method`, user do not have to specify a method when he changes the calculation type. In the `calculate` function of C-API, we can detect this and based on calculation type to change to a real default method as below:

* power flow: newton raphson
* state estimation: iterative linear
* short circuit: iec60909

Reply:
i agree. i will apply in this PR

Reply:
This only applies to C-API. In Python side, there is already a default keyword argument for calculaiton method per calculation type.